aiida/backends/general/migrations/utils.py

```diff
@@ -50,6 +50,16 @@ def put_object_from_string(uuid, name, content):
         handle.write(content)
 
 
+def del_object_from_repository(uuid, name):
```

Comment:
don't think you are actually using this, right? So could be removed

Reply:
Ah, yes, I set it for the backwards migrations. If you confirm those are not needed then I will delete it.

Reply:
no we don't support backwards migrations.